src/bridge/error/error_strings_test.cc

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <openssl/engine.h>
+#include <gtest/gtest.h>
+
+#include "src/bridge/error/error_strings.h"
+
+namespace kmsengine {
+namespace bridge {
+namespace error {
+namespace {
+
+using StatusCodeType = std::underlying_type<StatusCode>::type;
+using FunctionCodeType = std::underlying_type<FunctionCode>::type;
+
+TEST(ErrorStringsTest, PackReasonCodePacksIntoThirdArgument) {
```

Comment:
These tests are fine but pretty trivial. I'm wondering if a more useful test would be one that actually raises an error and checks that the expected string is printed. [`ERR_print_errors_fp`](https://www.openssl.org/docs/man1.1.0/man3/ERR_print_errors_fp.html) might help.

Reply:
Currently adding some with `ERR_func_error_string`, `ERR_reason_error_string`, etc.